dockers/docker-dhcp-relay/docker-dhcp-relay.supervisord.conf.j2

```diff
@@ -4,7 +4,7 @@ logfile_backups=2
 nodaemon=true
 
 [eventlistener:supervisor-proc-exit-listener]
-command=/usr/bin/supervisor-proc-exit-listener
+command=/usr/bin/supervisor-proc-exit-listener --container-name dhcp_relay
```

Comment:
Check builds are failing because you also need to reflect this change in the sample generated conf file used during unit tests: https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-config-engine/tests/sample_output/docker-dhcp-relay.supervisord.conf#L7

Reply:
Make a change in the config file /src/sonic-config-engine/tests/sample_output/docker-dhcp-relay.supervisord.conf.

Reply:
For pmon, do it need to make a same change?

Reply:
No. We currently do not have a unit test for that file.